.github/workflows/create-release-pr.yml

```diff
@@ -51,16 +54,19 @@ jobs:
           release-date: ${{ steps.set_variables.outputs.release_date }}
 
       - name: Create Pull Request
-        uses: peter-evans/create-pull-request@v4
-        with:
-          base: main
-          body: |
-            This PR updates the extension version and CHANGELOG.md.
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          echo "This PR updates the extension version and CHANGELOG.md.\n" >> pr-body.md
+          echo "Please close and reopen this PR to run the required workflows (Required statuses must pass before merging).\n" >> pr-body.md
+          echo "**The \`publish\` workflow will run automatically when this PR is merged.**" >> pr-body.md
 
-            Please close and reopen this PR to run the required workflows (Required statuses must pass before merging).
+          new_pr=$(cml pr create \
+            --token="${{ secrets.GITHUB_TOKEN }}" \
+            --message="Update version and CHANGELOG for release" \
+            --title="Update version and CHANGELOG for release" \
+            --body="$(cat ./pr-body.md)")
+          
+          gh pr edit $(echo "$new_pr" | awk -F / '{print $NF}') --add-label "make release go now"
```

Comment:
@dacbd the label isn't important, we just need to drop it from the if statement in `.github/workflows/publish.yml`